File: ruby/AzureSqlSample/connect.rb
    require 'tiny_tds'
@client = TinyTds::Client.new username: 'your_user@your_server', password: 'your_password',
    dataserver: 'your_server.database.windows.net', database: 'your_database', azure: true

puts 'Connecting to Azure SQL'

if @client.active? == true then puts 'Done' end

@client.close

File: nodejs/Unix-based/AzureSqlSample/connect.js
var Connection = require('tedious').Connection;
    var Request = require('tedious').Request;
    var TYPES = require('tedious').TYPES;

    // Create connection to database
    var config = {
      server: 'your_server.database.windows.net',	// update me
      authentication: {
          type: 'default',
          options: {
              userName: 'your_user', 			// update me
              password: 'your_password' 		// update me
          }
      },
      options: {
	  trustServerCertificate: true,
	  encrypt: true,
          database: 'your_database'			// update me
		
      }
    }
    var connection = new Connection(config);
    connection.connect();

    // Attempt to connect and execute queries if connection goes through
    connection.on('connect', function(err) {
      if (err) {
        console.log(err);
      } else {
        console.log('Connected');
      }
    });